packages/xrpl/test/models/batch.test.ts

```diff
@@ -0,0 +1,163 @@
+import { assert } from 'chai'
+
+import { validate } from '../../src'
+import { validateBatch } from '../../src/models/transactions/batch'
+import { assertTxValidationError } from '../testUtils'
+
+/**
+ * Batch Transaction Verification Testing.
+ *
+ * Providing runtime verification testing for each specific transaction type.
+ */
+describe('Batch', function () {
+  let tx: any
+
+  beforeEach(function () {
+    tx = {
+      Account: 'rJCxK2hX9tDMzbnn3cg1GU2g19Kfmhzxkp',
+      BatchSigners: [
+        {
+          BatchSigner: {
+            Account: 'rPMh7Pi9ct699iZUTWaytJUoHcJ7cgyziK',
+            SigningPubKey:
+              '02691AC5AE1C4C333AE5DF8A93BDC495F0EEBFC6DB0DA7EB6EF808F3AFC006E3FE',
+            TxnSignature:
+              '30450221008E595499C334127A23190F61FB9ADD8B8C501D543E37945B11FABB66B097A6130220138C908E8C4929B47E994A46D611FAC17AB295CFB8D9E0828B32F2947B97394B',
+          },
+        },
+      ],
+      Flags: 1,
+      RawTransactions: [
+        {
+          RawTransaction: {
+            Account: 'rJCxK2hX9tDMzbnn3cg1GU2g19Kfmhzxkp',
+            Amount: '5000000',
+            Destination: 'rPMh7Pi9ct699iZUTWaytJUoHcJ7cgyziK',
+            Fee: '0',
+            NetworkID: 21336,
+            Sequence: 0,
+            SigningPubKey: '',
+            TransactionType: 'Payment',
+          },
+        },
+        {
+          RawTransaction: {
+            Account: 'rPMh7Pi9ct699iZUTWaytJUoHcJ7cgyziK',
+            Amount: '1000000',
+            Destination: 'rJCxK2hX9tDMzbnn3cg1GU2g19Kfmhzxkp',
+            Fee: '0',
+            NetworkID: 21336,
+            Sequence: 0,
+            SigningPubKey: '',
+            TransactionType: 'Payment',
+          },
+        },
+      ],
+      TransactionType: 'Batch',
+    }
+  })
+
+  it('verifies valid Batch', function () {
+    assert.doesNotThrow(() => validateBatch(tx))
```

Comment:
the next statement encompasses this one, this is not useful.
```suggestion
```

Reply:
This one checks one with `BatchSigners`

Reply:
Yes, I understand that `validateBatch` also checks the `BatchSigners` field. 

However, `validate(tx)` invokes the respective `validateBatch(tx)` [here](https://github.com/XRPLF/xrpl.js/blob/a85a4507aff8acb349397cf70ceb8c13d268d9f5/packages/xrpl/src/models/transactions/transaction.ts#L277). So, I don't see the point of explicitly invoking the `validateBatch` method in the unit tests.